lib/nerves_hub_web/channels/device_channel.ex

```diff
@@ -34,7 +34,9 @@ defmodule NervesHubWeb.DeviceChannel do
 
   @decorate with_span("Channels.DeviceChannel.handle_info:after_join")
   def handle_info({:after_join, params}, %{assigns: %{device: device}} = socket) do
-    device = maybe_update_deployment(device)
+    device =
+      device
+      |> Deployments.verify_deployment_membership()
```

Comment:
```suggestion
    device = Deployments.verify_deployment_membership(device)
```
I think this is a bit cleaner.

Reply:
Agreed! Brain fart for me :P